samples/nrf9160/gps/src/main.c

```diff
@@ -128,324 +136,268 @@ static int activate_lte(bool activate)
 
 	return 0;
 }
+
+static void get_agps_data(struct k_work *item)
+{
+	ARG_UNUSED(item);
+
+	printk("\033[1;1H");
+	printk("\033[2J");
+	printk("New A-GPS data requested, contacting SUPL server, flags %d\n",
+			last_agps.data_flags);
+
+	activate_lte(true);
+	printk("Established LTE link\n");
+	if (open_supl_socket() == 0) {
```

Comment:
I see that checkpatch doesn't mind, but personally I prefer blank line before `if` statements unless a variable is assigned on the line before and checked in the `if`